appeals/web/src/server/appeals/appeal-details/representations/lpa-statement/lpa-statement.mapper.js

```diff
@@ -125,18 +126,16 @@ export function reviewLpaStatementPage(appealDetails, lpaStatement) {
 				{
 					value: COMMENT_STATUS.VALID,
 					text: 'Accept statement',
-					checked: lpaStatement?.status === COMMENT_STATUS.VALID
+					checked: 'acceptLPAStatement' in session
 				},
 				{
 					value: COMMENT_STATUS.VALID_REQUIRES_REDACTION,
-					text: 'Redact and accept statement',
-					checked: false // This status isn't persisted so will always be unchecked
+					text: 'Redact and accept statement'
 				},
 
 				{
 					value: COMMENT_STATUS.INCOMPLETE,
-					text: 'Statement incomplete',
-					checked: lpaStatement?.status === COMMENT_STATUS.INCOMPLETE
```

Comment:
Should this really never be checked? I think this was reported to me as a bug in my last ticket

Reply:
Yeah would you mind leaving this as it was so it doesn't conflict with this please? https://arc.net/l/quote/cwevgpio

Reply:
TY!